src/views/StandaloneDirectorsFiling.vue

```diff
@@ -673,6 +705,13 @@ export default {
       }
     },
 
+    async getErrorObj (errCode) {
+      const fetchUrl = this.payApiUrl + 'codes/errors/BCOL_ERROR'
+      const errObj = await axios.get(fetchUrl)
+      console.log(errObj)
+      return errObj.data
+    },
```

Comment:
Could use the mixin call instead to grab an error code so you wouldn't have to duplicate this function.
Also, it looks like BCOL_ERROR is hardcoded